docs/community/CODE_OF_CONDUCT.md

```diff
@@ -55,6 +55,14 @@ If any such relationship leads to harassment, the Coordinators and Mentors are n
 
 If you are not comfortable sharing any details, do see [Reporting Incidents](https://handbook.rcos.io/#/community/CODE_OF_CONDUCT?id=reporting-incidents), in addition to RPI resources: [Title IX Office@RPI](https://sexualviolence.rpi.edu/) and [Public Safety@RPI](https://publicsafety.rpi.edu/), for more information on how to escalate the situation to authorities.
 
+## Project Endorsement Regulations
+
+Students in RCOS that are already affiliated with / are contributors of a certain project may not use imperative statements to endorse joining their project. Outside of pitch season, contributing students may only answer questions concerning facts about their project and opinions about certain people in the project, but not about the project as a whole. During pitch season, they may only state their own opinions about their project. 
```

Comment:
Are the 2nd and 3rd sentences necessary? I'm not sure we need to restrict discussing entire projects, or only state opinion during pitch season. Seems a bit confusing to me.

Reply:
I agree. I think we should be more charitable and only ban discouraging other projects.

If project leads want more members, they have the freedom to pitch their project better than Shuttle Tracker does.